crates/biome_js_analyze/src/lint/nursery/no_leaked_render.rs

```diff
@@ -106,7 +106,7 @@ impl Rule for NoLeakedRender {
         let query = ctx.query();
         let model = ctx.model();
 
-        if !is_inside_jsx_expression(query.syntax()) {
+        if !is_inside_jsx_expression(query.syntax()).unwrap_or(false) {
```

Comment:
```suggestion
        if !is_inside_jsx_expression(query.syntax()).unwrap_or_default() {
```

It's more idiomatic to use the default value instead (which in this case is false). You use `unwrap_or` for not default values